packages/pdf-generator/src/utils/styles.js

```diff
@@ -94,6 +104,26 @@ export default (style = {}) => StyleSheet.create({
     defaultColor: {
         color: global_Color_light_300.value
     },
+    labelColorsLow: {
+        bgColor: global_palette_blue_50.value,
+        textColor: global_palette_blue_600.value,
+        iconColor: global_primary_color_100.value
+    },
+    labelColorsModerate: {
+        bgColor: global_palette_gold_50.value,
+        textColor: global_palette_gold_700.value,
+        iconColor: chart_global_warning_Color_200.value
+    },
+    labelColorsImportant: {
+        bgColor: '#fff5ec',
```

Comment:
does this not have a pf color? 🤔

Reply:
I believe that's a custom color. It's the same in the https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/Components/InsightsLabel/labels.scss